selfdrive/car/CARS_template.md

```diff
@@ -10,7 +10,7 @@ A supported vehicle is one that just works when you install a comma three. All s
 
 # {{all_car_info | length}} Supported Cars
 
-|{{Column | map(attribute='value') | join('|')}}|
+|{{Column | map(attribute='value') | join('|') | replace('Harness Kit', '<img width=2000>Harness Kit<img width=2000>')}}|
```

Comment:
what if we make a new function to get column name and pass in prefix/suffix? it might not be cleaner than this unfortunately, if so that's okay.